tests/x11/libqt5_qtbase.pm

```diff
@@ -30,26 +31,31 @@ sub run {
 
     select_console('x11');
     ensure_installed("libqt5-qttools yast2-installation", timeout => 180);
+    # we have at moment performance issue on aarch64 OSD only, so no need to change this for tumbleweed
+    # see poo#125873 for more details
+    if (is_aarch64 && is_sle) {
+        compile_qt5();
+    }
+    elsif (is_x86_64 || is_s390x || is_ppc64le || (is_aarch64 && is_tumbleweed)) {
+        designer_qt5();
+        release_notes();
+        compile_qt5();
+    }
+}
 
-    # Test designer-qt5
-    x11_start_program('designer-qt5');
-    wait_still_screen 3;
-    assert_and_click("designer-qt5-start");
-    wait_still_screen 3;
-    assert_screen("designer-qt5-main");
+sub designer_qt5 {
+    my $timeout = '90';
+    x11_start_program("designer-qt5", target_match => "designer-qt5-start", match_timeout => $timeout);
```

Comment:
Looking at poo#125873 and especially at https://openqa.suse.de/tests/10676638#step/libqt5_qtbase/25 I'm wondering, if increasing the timeout alone wouldn't be already enough?

Reply:
ok